lib/toBuffer.js

```diff
@@ -49,7 +49,8 @@ function toBuffer(value, encoding) {
             }
         }
     }
+
     throw new Error('[toBuffer] unsupported type in value. Use Buffer, string or Array');
 }
 
-module.exports = toBuffer;
\ No newline at end of file
+module.exports = toBuffer;
```

Comment:
Since there are no changes to the toBuffer file can you remove it from the pull request?